prefork_test.go

```diff
@@ -14,12 +14,12 @@ import (
 	"github.com/gofiber/fiber/v2/utils"
 )
 
+//nolint:paralleltest // TODO: Must be run sequentially due to global testPreforkMaster var.
 func Test_App_Prefork_Child_Process(t *testing.T) {
 	// Reset test var
 	testPreforkMaster = true
 
-	setupIsChild(t)
```

Comment:
why was this changed? no cleanup needed anymore? clean startup conditions would be good to avoid errors

Reply:
The teardown func was simply unsetting the env var, which is now taken care of by `t.Setenv`, thus dropping two funcs (`setupIsChild` and `teardownIsChild `) and making the code easier to read.

Reply:
yes correct, but the next session in another test is contaminated with it

what if the next tests uses the variables